doc/python/m.distance.py

```diff
@@ -25,7 +25,7 @@
 
 # %module
 # % label: Finds the distance between two or more points.
-# % description: If the projection is latitude-longitude, this distance is measured along the geodesic.
+# % description: If the projection is lat-long, distance measured along the geodesic.
```

Comment:
Please keep the original line and open an issue for this line. This is a general issue and we can't resolve it by less clear documentation which this shortening approach may eventually lead to.

Reply:
Got it, this is also something which I highlighted here in the existing `flake8` discussion: https://github.com/OSGeo/grass/issues/1522#issuecomment-2364560761
For this merge, I'll roll back this change

Reply:
Updated commit to remove this